src/ApiWrapper/index.js

```diff
@@ -10,7 +11,14 @@ const jsENV = whichJsEnv();
 export default ( () => {
   // await async dependencies
   // export the module
-  if (jsENV === 'browser') {
+  if (jsENV === 'browser' && process.env.REACT_APP_NODE_ENV === 'demo') {
```

Comment:
I wonder if it's possible to just either stick to one variable rather than two (jsEnv + react_app_node_env?
i.e web_demo, electron, production?

Reply:
possibly, could change how ` whichJsEnv` is implemented and check for demo there instead?

Reply:
I would probably leave it in here in the index as it's better to see that the API is being swapper in the API logic. If the demo is only going to exist as a browser, I would probably only check for demo, then assume it's browser and not have the 
`if (jsENV === 'browser' && process.env.REACT_APP_NODE_ENV === 'demo')`
but replace it with just checking for demo and if it's not demo, then it's either electron / browser (live)